conf/systemd.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
```

Comment:
Do we really need a systemd config for dokuwiki ?

Reply:
No, you are right. I wanted a system to be able to "start or stop" dokuwiki like a regular service from the admin panel in Yunohost. Maybe it is not doable or https://github.com/YunoHost/example_ynh/blob/master/scripts/install#L187 is more appropriate ?

Reply:
We're using systemd for app which effectively using a service. But, dokuwiki, as a php app, do not have any services, so there's no use of systemd for this app.
So, unfortunately, you can't start and stop dokuwiki as any service.

The command `yunohost service add` is used, as well, for app with a service. You can also remove it.

At least, that show us that the example app needs some extra comments to explain the different parts.

Reply:
Thanks for the explanation. I will remove them then.
I understand what the helper is doing from a technical point of view but not from a functionality one : "Do I need this part for the package? What feature it will add? Oh, it not needed for PHP app but needed for this kind of app"

Reply:
Well, `ynh_add_systemd_config` is a helper to help you configure a systemd config file, from an existing file in conf/.
But the helper will just help you configure a systemd service for an app which need one. You can usually find this information in the install procedure of the app itself.

`yunohost service add` is a yunohost cli command that linked a service to the admin interface, allowing then to stop, start and disable a service from the admin interface. Also, it gives a view to the log of the service.

But both of  these commands work only with apps which already using a systemd service.